ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -339,6 +346,7 @@ PrjFS_Result PrjFS_WritePlaceholderDirectory(
 
     if (mkdir(fullPath, 0777))
     {
+        LogWarning("PrjFS_WritePlaceholderDirectory: mkdir failed errno=%d stderror=%s", errno, strerror(errno));
```

Comment:
I just did some more digging and found that the `ENOENT` case is actually expected sometimes, see:

https://github.com/microsoft/VFSForGit/blob/aff5bb1f304afe326d218ccefc4b27c4e6795ae9/GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs#L1549

Could you move this `LogWarning` down to the `default` case?